index.js

```diff
@@ -2,7 +2,6 @@ const path = require("path");
 const sharp = require("sharp");
 const { version } = require("./package.json");
 const Vibrant = require("node-vibrant");
-var sortBy = require("lodash.sortby");
```

Comment:
I quite like this change.
May I suggest something? I'll move toPalette and tobase64 to their own module (for easier testing), write some test cases and either manually merge your PR or get you (if you have time only) to copy what you done to other file!

Reply:
Good idea move in a separate project; I want to use it inside my getFromUrl wrapper: https://github.com/Kikobeats/splashy/blob/master/index.js#L10

Reply:
now they are at least in their own file (module), I don't think they are worth a totally separate npm package! https://github.com/zouhir/lqip/blob/master/util.js

Reply:
would you like me to manually transfer your lodash removal fix to the `util` folder? I am cool to do so! not fair to get you top repeat your work! <3 thanks!